core/bin/zksync_server/src/main.rs

```diff
@@ -56,6 +56,9 @@ struct Cli {
     /// Path to the yaml with contracts. If set, it will be used instead of env vars.
     #[arg(long)]
     contracts_config_path: Option<std::path::PathBuf>,
+    /// Path to the yaml with contracts. If set, it will be used instead of env vars.
```

Comment:
Nit: AFAIU, this description is incorrect; gateway-related params are never parsed from env vars. Also, could be worth specifying "with *gateway* contracts".